csharp/Platform.Collections/Lists/IListExtensions.cs

```diff
@@ -343,6 +343,17 @@ public static int GenerateHashCode<T>(this IList<T> list)
             return hashAccumulator;
         }
 
+        /// <summary>
+        /// <para>Compares each list item to each other.</para>
+        /// <para>Сравнивает каждый элемент списка с друг другом.</para>
+        /// </summary>
+        /// <typeparam name="T"><para>The list's item type.</para><para>Тип элементов списка.</para></typeparam>
+        /// <param name="left"><para>The checked list.</para><para>Список для проверки.</para></param>
+        /// <param name="right"><para>The checked list.</para><para>Список для проверки.</para></param>
+        /// <returns>
+        /// <para>The relative position in the sort order.</para>
```

Comment:
Comment can be made using comment to `comparer.Compare` method from .NET documentation.

![image](https://user-images.githubusercontent.com/1431904/98445244-a32cde80-2127-11eb-9878-cc7533cddafc.png)

https://docs.microsoft.com/en-us/dotnet/api/system.collections.comparer.compare?view=netcore-3.1

You can click on Edit and get the XML of comments:

https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Collections/Comparer.xml#L213-L231

I have edited it for you:

```
<para>
	A signed integer that indicates the relative values of <paramref name="left" /> and <paramref name="right" /> lists' elements, as shown in the following table. 
	<list type="table">
		<listheader>
			<term>Value</term>
			<description>Meaning</description>
		</listheader>
		<item>
			<term>Is less than zero</term>
			<description>First non equal element of <paramref name="left" /> list is less than first not equal element of <paramref name="right" /> list.</description>
		</item>
		<item>
			<term>Zero</term>
			<description>All elements of <paramref name="left" /> equals to all elements of <paramref name="right" />.</description>
		</item>
		<item>
			<term>Is greater than zero</term>
			<description>First non equal element of <paramref name="left" /> list is greater than first not equal element of <paramref name="right" /> list.</description>
		</item>
	</list>
</para>
```

The same can be done for Russian version.